main/boost/APKBUILD

```diff
@@ -45,27 +45,35 @@ _libs="
 	filesystem
 	graph
 	iostreams
+	locale
+	log_setup
+	log
 	math
 	prg_exec_monitor
 	program_options
 	python3
-	python
+	python2
 	random
 	regex
 	serialization
 	signals
+	stacktrace_basic
+	stacktrace_noop
 	system
 	thread
+	timer
+	type_erasure
 	unit_test_framework
 	wave
 	wserialization
 	"
 for _lib in $_libs; do
-	case "$_options_carch" in
-		*$_lib*) continue;;
-	esac
+	for _option in $_options_carch; do
+		case "$_option" in --without-${_lib}) continue ;; esac
+	done
```

Comment:
why is this needed? What is broken with how it was?

Reply:
Dropping the `--without-` part is the main problem.